src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,11 +570,16 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
I don't know enough about what's happening here to say. @lefticus or @kbenne might have a better idea. I would expect the MinGW builds to follow the same pattern as the Linux GCC builds.